crates/utils/src/settings/mod.rs

```diff
@@ -13,8 +13,16 @@ use structs::{DatabaseConnection, PictrsConfig, PictrsImageMode, Settings};
 static DEFAULT_CONFIG_FILE: &str = "config/config.hjson";
 
 pub static SETTINGS: Lazy<Settings> = Lazy::new(|| {
+  if let Ok(value) = env::var("LEMMY_INITIALIZE_WITH_DEFAULT_SETTINGS") {
+    if value.eq("true") {
```

Comment:
Should be enough with `env::var("LEMMY_INITIALIZE_WITH_DEFAULT_SETTINGS").is_ok()`, no need to check for a specific value.

Reply:
Its also how we handle other env vars eg [LEMMY_TEST_FAST_FEDERATION](https://github.com/LemmyNet/lemmy/blob/main/crates/federate/src/util.rs#L31). I dont see any reason why someone would try to disable it by setting false, instead you would `unset` it.

Reply:
ok cool